components/content/content.scss

```diff
@@ -18,15 +18,22 @@ a:hover {
 .links {
     opacity: 0;
     animation: onLoad 0.5s ease-out forwards;
+
+    &__link {
+        margin-bottom: 24px;
+
+        &__title:hover{
```

Comment:
@selbekk Her fungerer ikke hover-stylingen på tittelen (a-tag). Hvis jeg legger det til på hele linken (&__link:hover) så fungerer det, men da på tvers av hele siden fordi listeelementet tar opp hele bredden i nettleseren. Vet du hvorfor det ikke fungerer på a-tagen?

Reply:
Denne var tricky! Men [Stack Overflow](https://stackoverflow.com/questions/20022097/css-animation-not-working-for-a-tag) hadde svaret!

Det er fordi det ikke går an å animere elementer med `display: inline`. Du må sette `display: inline-block;` for at det skal fungere - det burde ikke påvirke noe annet.